distribution/lib/Standard/Base/0.0.0-dev/package.yaml

```diff
@@ -10,16 +10,8 @@ maintainers:
     email: contact@enso.org
 component-groups:
   new:
-    - Input:
-        exports:
-          - Standard.Base.System.File.File.new
-          - Standard.Base.Data.read
-          - Standard.Base.Data.read_text
-          - Standard.Base.Data.list_directory
-    - Web:
-        exports:
-          - Standard.Base.Network.HTTP.HTTP.new
-          - Standard.Base.Data.fetch
+    - Input: {}
+    - Web: {}
```

Comment:
Do the groups still need to be defined in `package.yaml`?

What if we annotate a method with a `GROUP xyz` when `xyz` is not defined in `package.yaml`?

Reply:
Yes the groups have to be defined in the package.yaml.
It is supposed to generate an error but that may not be done yet,

Reply:
IDE will print the error log and the entry will not be assigned to any group.